bitcoin/src/blockdata/script/push_bytes.rs

```diff
@@ -293,23 +293,51 @@ impl PushBytes {
 
     /// Decodes an integer in script(minimal CScriptNum) format.
     ///
+    /// This code is based on the `CScriptNum` constructor in Bitcoin Core (see `script.h`).
```

Comment:
We could link to the file?

Example can be seen in `blockdata/script/build.rs`

```rust
/// [CheckMinimalPush]: <https://github.com/bitcoin/bitcoin/blob/99a4ddf5ab1b3e514d08b90ad8565827fda7b63b/src/script/script.cpp#L366>
```

Note there are a ton of URLs in the codebase that don't have `<>` on them, I believe this is wrong. I thought clippy caught it but perhaps not.